### ISSUE TITLE
Do not show Clipboard contains markdown  warning in same document

### ISSUE THREAD

**Issue by holgerbrandl** [vsch/idea-multimarkdown#786]
It should not show the warning `Clipboard contains markdown links that may need to be adjusted` when copying and pasting bits and pieces around within the __same__ document. Obviously there's nothing to be adjusted in such a case.

**Comment by vsch** [comment]
@holgerbrandl, I think it is a bug I introduced when I fixed new Chrome link detection by loosening clipboard data types. It was too loose and the code triggers on non HTML. I remember fixing it in the dev version but will verify.

Unfortunately I am at least a week before I can even think of making an EAP release. The 2019.3 Beta threw me for a loop because of bundled Kotlin 1.30.60. An old library fails at runtime. I stopped bundling Kotlin runtime with the plugin because the IDEs have it so why add the extra payload. Seems it was a shortsighted decision.

I am in the process of getting the library's replacement ready but it will take some time to get all that code changed. Probably over 500 files with some code adjusted to the quirks of that library so replacement will have to be done carefully in places.

I was planning to do it for the last 2 years but kept putting it off because of the effort involved. Now, I have to do it quickly. It will be worth it. Just the paragraph wrapping replacement makes a noticeable difference in typing response. The code will be more stable and maintainable. Much faster and less memory footprint. Tons of caching with weak refs for cached elements to make sure the stuff does not hang around longer than needed.

I promise that I will make an EAP release as soon as possible.
